test/testrunner.py

```diff
@@ -132,9 +145,12 @@ def main():
         # Pull images for necessary tests
         if specific_test_type == "sanity":
             pull_dlc_images(all_image_list)
-        if specific_test_type == "eks" and not is_all_images_list_eia :
-            frameworks_in_images = [framework for framework in ("mxnet", "pytorch", "tensorflow")
-                                    if framework in dlc_images]
+        if specific_test_type == "bai":
+            build_bai_docker_container()
```

Comment:
How long does the build take? Why do we need to build the container on every test run? We should have a better mechanism to re-use an existing container. Also, if you're building a container image: it's better to have a copy in ECR. To speed up the build every time: first pull the image (so that it's cached), and then build+push again. The build is faster.

Reply:
The build itself happens in less than two minutes. The container is using a pre-existing one and adding on uuid and curl.  Once there is a easily available container with these and bash 5.0 this can easily be changed.

Reply:
@nskool this way we get the latest version of the conda container each time -- there are only two packages installed on top of the container.